webboot/webboot.go

```diff
@@ -8,9 +8,42 @@
 package main
 
 import (
-	"fmt"
+	"flag"
+	"log"
+	"os"
+
+	"github.com/u-root/webboot/pkg/mountkexec"
+)
+
+type distro struct {
+	kernel       string
+	initrd       string
+	cmdline      string
+	downloadLink string
+}
+
+var (
+	cmd       = flag.String("cmd", "", "Command Line")
+	mountDir  = flag.String("dir", "mountDir", "The mount point of the ISO")
+	osSystems = map[string]distro{
+		"TinyCore": distro{"/boot/vmlinuz64 ", "/boot/initrd", "--reuse-cmdline", "https:louis.com"},
+	}
 )
 
 func main() {
-	fmt.Println("weboot up")
+
+	flag.Parse()
+	if len(flag.Args()) < 1 {
+		log.Fatal("error:pass in a valid operating system")
+		os.Exit(1)
```

Comment:
you don't need an os.Exit here. log.Fatal exits.
in general you should provide a usage() function and call that for any errors.

Reply:
How do I implement this usage function?